HomepageV2.css

```diff
@@ -0,0 +1,335 @@
+    font-size: 15px;
+    right: 40px;
+    top: 31px;
+    z-index: 100;
+    height: 16px;
+    overflow: hidden;
+    opacity: 1;
+    transition: 0.2s ease-in-out;
+}
+
+.contactText2 {
+    position: absolute;
+    float: right;
+    color: rgb(249, 211, 139);
+    font-size: 15px;
+    right: 40px;
+    top: 31px;
+    z-index: 100;
+    height: 0;
+    overflow: hidden;
+    opacity: 0;
+    transition: 0.2s ease-in-out;
+}
+
+.contactInfo:hover .contactText1 {
+    opacity: 0;
+    height: 0;
+}
+
+.contactInfo:hover .contactText2 {
+    opacity: 1;
+    height: 16px;
+}
+
+#sidenav {
+    position: absolute;
+    top: 33%;
+    left: 35px;
+    z-index: 100;
+    transition: 0.5s ease-in-out;
+}
+
+.sidenavContent {
+    line-height: 30px;
+}
+
+.sidenavContent a {
+    text-decoration: none;
+    color: rgb(249, 211, 139);
+    font-size: 15px;
+    opacity: 0.4;
+    transition: 0.2s ease-in-out;
+}
+
+.sidenavContent a:hover {
+    opacity: 1;
+    font-size: 16px;
+}
+
+.sidenavContent .activeLink {
+    opacity: 1;
+    font-size: 16px;
+}
+
+.sidenavContent td {
+    padding: 0;
+    width: 0px;
+    height: 25px;
+    overflow: hidden;
+    display: block;
+    transition: 0.5s ease-in-out;
+}
+
+.largeTitle {
+    margin-top: 20px;
+    position: relative;
+    font-size: 70px;
+    color: rgb(249, 211, 139);
+    text-align: center;
+    z-index: 100;
+    font-family: "neutra";
+}
+
+.thinTitle {
+    font-family: 'robot', sans-serif;
+    margin-top: -45px;
+    margin-right: 70px;
+    position: relative;
+    font-size: 240px;
+    text-align: center;
+    z-index: 100;
+    font-weight: light;
+    opacity: 0.6;
+}
+
+.robotImage {
+    margin-top: -317px;
+    position: relative;
+    z-index: 100;
+    opacity: 0.7;
+    width: 245px;
+    height: 245px;
+}
+.background1 {
+    background-color:rgb(170, 7, 35);
+    padding: 0;
+    margin: 0;
+    height: 200%;
+    width: 52%;
+    position: absolute;
+    z-index: 0;
+}
+.background2 {
+    background-color: rgb(137, 4, 29);
+    padding: 0;
+    margin: 0;
+    margin-left: 50%;
+    height: 200%;
+    width: 50%;
+    position: absolute;
+    z-index: 0;
+}
+
+.texture {
+    background-image: url("images/background-texture.png");
+    background-size: 100px 100px;
+    opacity: 0.6;
+    width: 100%;
+    height: 100%;
+    position: absolute;
+}
+
+.topnav {
+    z-index: 100;
+}
+
+#topButton {
+    padding:10px;
+    background-color:rgb(249, 211, 139);
+    position:fixed;
+    bottom:25px;
+    right:25px;
+    z-index:100;
+    border-radius:50%;
+    opacity:0;
+    bottom:-50px;
+    transition:0.3s ease-in-out;
+}
+
+/* Line stuff */
+
+.line {
+    margin: auto;
+    height: 2px;
+    background-color: rgb(249, 211, 139);
+    opacity: 0;
+    width: 0;
+    transition-duration: 0.25s;
+    border-radius: 2px;
+    margin-left: 35px;
+}
+
+#toggleMenu:hover .line {
+    opacity: 1;
+    width: 42px;
+}
+
+
+/* Hamburger stuff */
+
+.hamburger {
```

Comment:
Put this hamburger stuff in a different .css file? I assume you are going to reuse a lot of this